index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <title>Todo</title>
+
+    <meta charset="UTF-8">
+    <meta name="description" content="Very simple todo list">
+    <meta name="author" content="Shamim">
+    <meta name="viewport" content="width=device, initial-scale=1.0">
+
+    <link rel="stylesheet" href="./assests/css/index.css">
+    <script src="https://kit.fontawesome.com/ccf39592f4.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+    <main class="main-box">
+      <h1 class="title">Todo</h1>
+      <nav>
+        <ul class="nav-list-box">
+          <li><p>All Tasks 1</p></li>
+          <li><p>Completed 2</p></li>
+          <li><p>Uncompleted 3</p></li>
+        </ul>
+      </nav>
+      <ul class="add-task-box">
+        <li><input class="add-task-input" placeholder="Write here..."></li>
+        <li><i class="fa-solid fa-plus add-task-icon"></i></li>
+      </ul>
+      <ul class="task-list-box">
+        <li class="task-box-wrapper">
+          <ul class="task-box">
```

Comment:
ul and li are extra here. the todo list is a ul and each todo is a li but inside li you can have p and i tags directly.

Reply:
thaaaaanks